live_validation_scripts/validate_live_pixel.mjs

```diff
@@ -3,61 +3,578 @@
+
+        for (const [name, def] of Object.entries(pixelsDefs)) {
+            pixelMap.set(name, {
+                owners: def.owners || [],
+                documented: true,
+                numPasses: 0,
+                numFailures: 0,
+                numAppVersionOutOfDate: 0,
+                numAccesses: 0,
+                sampleErrors: [],
+            });
+            stats.numPixelDefinitions++;
+        }
+
+        getPixelOwners(pixelsDefs).forEach((pixel) => {
+            if (!userMap[pixel.owner]) {
+                console.log(`WARNING: Invalid pixel owner: ${pixel.owner} for pixel: ${pixel.name}`);
+            } else {
+                allPixelOwners.add(pixel.owner);
+
+                if (ownerMap.has(pixel.owner)) {
+                    const existingEntry = ownerMap.get(pixel.owner);
+                    if (!existingEntry.pixels.includes(pixel.name)) {
+                        existingEntry.pixels.push(pixel.name);
+                    }
+                } else {
+                    ownerMap.set(pixel.owner, {
+                        asanaId: userMap[pixel.owner].asanaId,
+                        pixels: [pixel.name],
+                    });
+                }
+            }
+        });
+    });
+
+    stats.numValidOwners = ownerMap.size;
+    console.log(
+        `Processed ${stats.numPixelDefinitionFiles} pixel definition files with a total of ${stats.numPixelDefinitions} pixel definitions.`,
+    );
+    console.log(`Found ${stats.numValidOwners} unique owners in pixel definitions.`);
+}
+
+async function validateLivePixels(mainDir, csvFile) {
     console.log(`Validating live pixels in ${csvFile} against definitions from ${mainDir}`);
 
-    const productDef = fileUtils.readProductDef(mainDir);
-    const experimentsDef = fileUtils.readExperimentsDef(mainDir);
-    const commonParams = fileUtils.readCommonParams(mainDir);
-    const commonSuffixes = fileUtils.readCommonSuffixes(mainDir);
-    const tokenizedPixels = fileUtils.readTokenizedPixels(mainDir);
+    console.log('mainDir:', mainDir);
+
+    // This will be equal to stats.documentedPixels at the end of validation
+    console.log(`pixelMap size at start of validation/num documented pixels: ${pixelMap.size}`);
+
+    let productDef = {};
+    let experimentsDef = {};
+    let commonParams = {};
+    let commonSuffixes = {};
+    let pixelIgnoreParams = {};
+    let globalIgnoreParams = {};
+
+    try {
+        productDef = fileUtils.readProductDef(mainDir);
+        experimentsDef = fileUtils.readExperimentsDef(mainDir);
+        commonParams = fileUtils.readCommonParams(mainDir);
+        commonSuffixes = fileUtils.readCommonSuffixes(mainDir);
+
+        pixelIgnoreParams = fileUtils.readIgnoreParams(mainDir);
+
+        globalIgnoreParams = fileUtils.readIgnoreParams(fileUtils.GLOBAL_PIXEL_DIR);
+    } catch (error) {
+        console.error('Error reading input files:', error);
+        process.exit(1);
+    }
```

Comment:
The standard in this repo is to just err out instead of trying to catch all possible exceptions (we're not running in an environment where an exception will cause a major failure). So I'd keep this as it was for readability and the opportunity to use `const`.